src/python/CMakeLists.txt

```diff
@@ -415,6 +415,9 @@ if(PYTHONINTERP_FOUND)
       add_gudhi_py_test(test_dtm)
     endif()
 
+    # Weighted Rips
+    add_gudhi_py_test(test_weighted_rips)
```

Comment:
I think there is at least one more place in this file where the new module should be mentioned.